### DISCUSSION TITLE
请问alist是否可以添加已经使用rclone加密的存储？

### DISCUSSION THREAD

**Discussion by gdtv** [AlistGo/alist#6798] [Q&A] [answered]
您好，我以前用rclone加密了一个webdav存储，rclone设置如下：

```
[local]
type = local

[my-disk]
type = webdav
url = https://xxxxxxx.com
user = xxxx
pass = xxxx

[my-disk-crypt]
type = crypt
remote = my-disk:/
filename_encryption = standard
directory_name_encryption = true
password = xxxxxxxxxxxx
password2 = xxxxxxxxxxxxxx
```
我现在想改用alist。

因为这里面的文件太多，我无法先下载回来再用alist上传。

是否有办法直接将这个存储添加到alist里面呢？

希望是可以直接用alist添加存储，而不是先用rclone挂载到本地，然后再用alist添加本地存储。

谢谢。

**Comment by pongfcnkl** [comment, marked as answer]
可以 参考https://github.com/alist-org/alist/issues/5544
![截图_20240716195101](https://github.com/user-attachments/assets/44420cfe-cbbe-4efd-b382-82b7b83fee92)
